packages/backend/src/database/entities/projects.ts

```diff
@@ -37,3 +37,23 @@ export type ProjectTable = Knex.CompositeTableType<
     CreateDbProject,
     UpdateDbProject
 >;
+
+type DbCachedExplores = {
+    project_uuid: string;
+    explores: any;
+};
+
+export type CachedExploresTable = Pick<
+    DbCachedExplores,
+    'project_uuid' | 'explores'
+>;
+
+type DbCachedWarehouse = {
+    project_uuid: string;
+    warehouse: any;
+};
+
+export type CachedWarehouseTable = Pick<
```

Comment:
Yes, I didn't know about this file, cool. Just curious, what's the impact in the rest of the code ?

Reply:
For `knex` queries it knows the type returned by simple queries and inserts rather than using `any`. Makes it more likely that typescript will throw errors if we pass the wrong data into insert queries / use wrong types after selecting data